modules/virt-checking-storage-configuration.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-running-cluster-checkups.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-checking-storage-configuration_{context}"]
+= Running a storage checkup
+
+Use a predefined checkup to verify that the {product-title} cluster storage is configured optimally to run {VirtProductName} workloads.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `ServiceAccount`, `Role`, and `RoleBinding` manifest for the storage checkup:
++
+.Example service account, role, and rolebinding manifest file
+[%collapsible]
+====
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: storage-checkup-sa
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: storage-checkup-role
+rules:
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs: ["get", "update"]
+  - apiGroups: [ "kubevirt.io" ]
+    resources: [ "virtualmachines" ]
+    verbs: [ "create", "delete" ]
+  - apiGroups: [ "kubevirt.io" ]
+    resources: [ "virtualmachineinstances" ]
+    verbs: [ "get" ]
+  - apiGroups: [ "subresources.kubevirt.io" ]
+    resources: [ "virtualmachineinstances/addvolume", "virtualmachineinstances/removevolume" ]
+    verbs: [ "update" ]
+  - apiGroups: [ "kubevirt.io" ]
+    resources: [ "virtualmachineinstancemigrations" ]
+    verbs: [ "create" ]
+  - apiGroups: [ "cdi.kubevirt.io" ]
+    resources: [ "datavolumes" ]
+    verbs: [ "create", "delete" ]
+  - apiGroups: [ "" ]
+    resources: [ "persistentvolumeclaims" ]
+    verbs: [ "delete" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: storage-checkup-role
+subjects:
+  - kind: ServiceAccount
+    name: storage-checkup-sa
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: storage-checkup-role
+----
+====
+
+. Apply the `ServiceAccount`, `Role`, and `RoleBinding` manifest:
```

Comment:
You may also mention the prerequisite `ClusterRoleBinding` where cluster admin should create the required `cluster-reader` permissions for dedicated `storage-checkup-sa` `ServiceAccount` and namespace, as mentioned [here](https://github.com/kiagnose/kubevirt-storage-checkup?tab=readme-ov-file#permissions).